web/src/features/moreCast2/interfaces.ts

```diff
@@ -24,17 +24,26 @@ export interface BaseRow {
   stationCode: number
   stationName: string
   forDate: DateTime
+  latitude: number
+  longitude: number
 }
 
 export interface MoreCast2Row extends BaseRow {
   // Fire weather indices
-  ffmc: number
-  dmc: number
-  dc: number
-  isi: number
-  bui: number
-  fwi: number
-  dgr: number
+  ffmcCalcActual: number
+  dmcCalcActual: number
+  dcCalcActual: number
+  isiCalcActual: number
+  buiCalcActual: number
+  fwiCalcActual: number
+  dgrCalcActual: number
+  ffmcCalcForecast?: PredictionItem
```

Comment:
Ya basically. Since the FWI values fit in well being categorized as Actuals and Forecasts (I think), I followed the patten of rendering actuals and forecast cells.
I initially tried to type them as numbers and I'm honestly having a hard time completely remembering why that way didn't work well....